R/callouts.R

```diff
@@ -45,13 +45,12 @@ callouts.gsplot <- function(object, ..., side=c(1,2)){
 #' 
 #' @rdname callouts
 #' @export
-callouts.default <- function(x, y=NULL, labels=NA, length=0.1, angle=30, ...){
+callouts.default <- function(x, y=NULL, labels=NA, length=0.1, angle='auto', ...){
   
   x <- x[!is.na(labels)]
   y <- y[!is.na(labels)]
   labels <- labels[!is.na(labels)]
   
-  stopifnot(angle >= 0, angle <= 360)
```

Comment:
would still like to check this, in case the angle comes in as -45, we should fail. Just need to test in a different way.